acme/acme/client.py

```diff
@@ -245,26 +245,34 @@ def answer_challenge(self, challb, response):
 
     @classmethod
     def retry_after(cls, response, default):
-        """Compute next `poll` time based on response ``Retry-After`` header.
+        """Compute next `poll` time based on response ``Retry-After`` header,
+        per https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.37
```

Comment:
First docstring paragraph should be limited to one line only, please move the link somewhere down.